packages/main/src/MultiComboBox.ts

```diff
@@ -1106,22 +1105,20 @@ class MultiComboBox extends UI5Element implements IFormInputElement {
 	}
 
 	_onItemTab() {
-		this._inputDom.focus();
```

Comment:
why is this focus handling removed?

Reply:
It was out of place and with no impact on the functionality, on item TAB press the focus is dispatched to the next focusable element.